oregon/static/js/or2.json

```diff
@@ -0,0 +1 @@
,12],[10,29],[3,39],[3,31],[4,25],[4,16],[5,13],[5,-3],[-2,-10],[-7,-72],[20,25],[16,38],[16,29],[-45,35],[-20,45],[1,100],[15,51],[5,76],[7,21],[4,-74],[8,-61],[21,-12],[11,-27],[28,-18],[-5,16],[3,13],[6,13],[4,15],[-2,17],[-5,30],[-10,11],[-4,33],[6,51],[-36,-14],[-28,34],[-12,17],[6,35],[10,71],[3,64],[26,33],[-1,50],[4,13],[10,5],[11,-10],[10,-8],[7,-2],[10,8],[11,27],[-8,-4],[-6,-2],[-5,-3],[-14,4],[-8,12],[-5,17],[-12,-8],[-4,-18],[-12,-8],[-2,-28],[0,-31],[-7,-19],[-2,17],[3,74],[-5,65],[-8,21],[-20,19],[-7,21],[2,19],[-5,4],[-11,-5],[-1,17],[17,33],[2,33],[9,47],[3,135],[-4,56],[-12,51],[-26,45],[10,24],[12,11],[11,33],[16,13],[21,17],[7,30],[6,51],[-6,54],[-11,105],[-17,90],[-49,189],[-39,71],[6,11],[18,5],[18,-26],[-8,-15]],[[1393,9916],[-5,-17],[-15,12],[-19,15],[-12,26],[-5,31],[2,16],[11,-3],[14,-17],[20,-27],[9,-36]],[[1590,6037],[9,0],[0,4],[-9,0],[0,-4]],[[3750,8715],[11,0],[0,3],[-11,0],[0,-3]],[[7989,7810],[11,0],[0,3],[-11,0],[0,-3]],[[6178,7882],[9,0],[0,3],[-9,0],[0,-3]],[[5884,6657],[6,0],[0,3],[-6,0],[0,-3]],[[3328,1355],[10,0],[0,4],[-10,0],[0,-4]],[[4573,5404],[9,0],[0,3],[-9,0],[0,-3]],[[1813,6173],[8,0],[0,4],[-8,0],[0,-4]],[[2232,7585],[7,0],[0,3],[-7,0],[0,-3]],[[1542,6859],[6,0],[0,3],[-6,0],[0,-3]],[[1182,7190],[13,0],[0,3],[-13,0],[0,-3]],[[2161,8176],[11,0],[0,3],[-11,0],[0,-3]],[[2010,8291],[9,0],[0,4],[-9,0],[0,-4]],[[6797,3727],[6,0],[0,4],[-6,0],[0,-4]],[[9031,4662],[4,0],[0,3],[-4,0],[0,-3]],[[9358,4760],[8,0],[0,4],[-8,0],[0,-4]],[[6926,5623],[12,0],[0,3],[-12,0],[0,-3]],[[9120,6503],[9,0],[0,3],[-9,0],[0,-3]],[[7853,6446],[9,0],[0,3],[-9,0],[0,-3]],[[9045,7877],[7,0],[0,3],[-7,0],[0,-3]],[[8262,7538],[6,0],[0,4],[-6,0],[0,-4]],[[8985,8049],[11,0],[0,3],[-11,0],[0,-3]],[[7088,8634],[11,0],[0,3],[-11,0],[0,-3]],[[4184,8443],[11,0],[0,4],[-11,0],[0,-4]],[[4241,6199],[8,0],[0,3],[-8,0],[0,-3]],[[4017,4831],[6,0],[0,4],[-6,0],[0,-4]],[[5196,451],[10,0],[0,3],[-10,0],[0,-3]],[[3442,516],[14,0],[0,4],[-14,0],[0,-4]],[[2303,449],[9,0],[0,3],[-9,0],[0,-3]],[[2113,793],[9,0],[0,4],[-9,0],[0,-4]],[[1971,736],[12,0],[0,3],[-12,0],[0,-3]],[[1523,1021],[12,0],[0,3],[-12,0],[0,-3]],[[188,2619],[8,0],[0,3],[-8,0],[0,-3]],[[1511,2859],[10,0],[0,3],[-10,0],[0,-3]],[[413,3240],[10,0],[0,4],[-10,0],[0,-4]],[[926,3887],[11,0],[0,3],[-11,0],[0,-3]],[[1861,4220],[15,0],[0,3],[-15,0],[0,-3]],[[1906,4808],[11,0],[0,3],[-11,0],[0,-3]],[[1826,4822],[8,0],[0,3],[-8,0],[0,-3]],[[776,6159],[7,0],[0,4],[-7,0],[0,-4]],[[1889,6914],[7,0],[0,4],[-7,0],[0,-4]],[[2417,7888],[13,0],[0,3],[-13,0],[0,-3]],[[2332,8278],[8,0],[0,4],[-8,0],[0,-4]],[[890,8121],[11,0],[0,3],[-11,0],[0,-3]],[[2159,9064],[11,0],[0,3],[-11,0],[0,-3]],[[934,9840],[7,0],[0,3],[-7,0],[0,-3]],[[130,2093],[9,0],[0,4],[-9,0],[0,-4]],[[4289,7455],[8,0],[0,4],[-8,0],[0,-4]],[[1651,6697],[11,0],[0,4],[-11,0],[0,-4]],[[4070,6493],[15,0],[0,3],[-15,0],[0,-3]]],"transform":{"scale":[0.0008112574283090495,0.00042568292475525147],"translate":[-124.56676184799996,41.99954022137224]}}
```

Comment:
I used gdal, topojson, and a public-domain geo dataset. Since only the state-level data is being used  (no city data) from this file at this point, it should not be necessary to touch it (at least until the big one hits and the shape of Oregon changes).